prepare-videochat.sh

```diff
@@ -130,6 +137,9 @@ HEIGHT=480
 # Frame rate of video
 GST_FPS=5
 
+# Choose audio codec from wav, aac or opus
+AUDIO_CODEC=opus
```

Comment:
No, opus is fine - just make the changes to AUDIO_CODEC in a different
commit, to make the change more explicit to someone who is skimming through
the history :-). Thanks!

On 14 June 2016 at 23:34, Developer Laptander <notifications@github.com>
wrote:

> In prepare-videochat.sh
> <https://github.com/bluezio/ipwebcam-gst/pull/29#discussion_r67070386>:
>
> > @@ -130,6 +137,9 @@ HEIGHT=480
> >  # Frame rate of video
> >  GST_FPS=5
> >
> > +# Choose audio codec from wav, aac or opus
> > +AUDIO_CODEC=opus
>
> You want me to change default codec back to wav or just remove this
> variable?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bluezio/ipwebcam-gst/pull/29/files/71c6987ce6fd7ffee7db2011ff7014c2ae51dd8e#r67070386>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAC1qHrYHgKqxsDM8n5g7Fvu8oguKGuLks5qLyyOgaJpZM4Iz64H>
> .
>